test/functional/wallet_listtransactions.py

```diff
@@ -208,5 +210,32 @@ def get_unconfirmed_utxo_entry(node, txid_to_match):
         assert_equal(self.nodes[0].gettransaction(txid_3b)["bip125-replaceable"], "no")
         assert_equal(self.nodes[0].gettransaction(txid_4)["bip125-replaceable"], "unknown")
 
+        self.log.info('Check "coin-join" transaction')
+        input_0 = next(i for i in self.nodes[0].listunspent(query_options={'minimumAmount': 0.2}, include_unsafe=False))
+        input_1 = next(i for i in self.nodes[1].listunspent(query_options={'minimumAmount': 0.2}, include_unsafe=False))
+        raw_hex = self.nodes[0].createrawtransaction(
+            inputs=[
+                {
+                    'txid': input_0['txid'],
+                    'vout': input_0['vout'],
+                },
+                {
+                    'txid': input_1['txid'],
+                    'vout': input_1['vout'],
+                },
+            ],
+            outputs={
+                self.nodes[0].getnewaddress(): 0.123,
+                self.nodes[1].getnewaddress(): 0.123,
+            },
+        )
+        raw_hex = self.nodes[0].signrawtransactionwithwallet(raw_hex)['hex']
+        raw_hex = self.nodes[1].signrawtransactionwithwallet(raw_hex)['hex']
+        txid_join = self.nodes[0].sendrawtransaction(hexstring=raw_hex, maxfeerate=0)
+        fee_join = self.nodes[0].getmempoolentry(txid_join)['fees']['base']
+        #assert_equal(fee_join, self.nodes[0].gettransaction(txid_join)['fee'])
```

Comment:
hm?

Reply:
That's the bug. the `#` can be removed when the bug is fixed

Reply:
Thanks, fixed comment to clarify